jetty-ee9/jetty-ee9-tests/jetty-ee9-test-client-transports/src/test/java/org/eclipse/jetty/ee9/test/client/transport/HttpClientContinueTest.java

```diff
@@ -738,41 +733,56 @@ public void testExpect100ContinueWithTwoResponsesInOneRead(Transport transport)
         }
     }
 
-    @ParameterizedTest
-    @MethodSource("transports")
-    public void testNoExpectRespond100Continue(Transport transport) throws Exception
+    @Test
+    public void testNoExpectRespond100Continue() throws Exception
     {
-        start(transport, new HttpServlet()
+        startClient(Transport.HTTP);
```

Comment:
No HTTPS testing anymore?

Reply:
No, because the code was not testing HTTPS, but HTTP clear text twice, so once is enough.